code/src/index.css

```diff
@@ -1,13 +1,419 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: 'sora', 'work sans', 'Helvetica Neue', 'helvetica', arial,
     sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    --background: #FBF8F3;
+    --primary: #232323;
+    --secondary: #FF5733;
+    --third: #1DB954;
+   background: var(--background);
+   color: var(--color);
+   
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+
+.main-container {
+  width: 100%;
+  padding-bottom: 20px;
+}
+
+/*HEADER*/
+.header-container {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin: 40px 20px 0 20px;
+  flex-wrap: wrap;
+}
+
+
+.header-title {
+  width: 100%;
+  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+  background-size:  400% 400%;
+  animation: gradient 15s ease infinite;
+  background-clip: text;
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent; 
+  -moz-text-fill-color: transparent;
+  font-size: 40px;
+  font-family: 'Sora', sans-serif;
+  font-weight: 800;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+
+/*ALBUM CONTAINER*/
+.album-container {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 60px;
+  margin-top: 20px;
+  
+}
+ 
+ .album-card {
+    width: 89%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 0px;
+    background: var(--background);
+ }
+
+ .album-name{
+  font-weight: 800;
+  font-size: 18px;
+  color: var(--primary);
+  text-decoration: none;
+  padding-top: 30px;
+  text-transform: uppercase;
+ }
+
+ .album-name:hover{
+  color: var(--third);
+  text-transform: uppercase;
+ }
+ 
+ .artists > a:link {
+   display: inline-flex;
+   font-size: 16px;
+   font-weight: 500;
+   color:  var(--primary);
+   opacity: 0.72;
+   text-decoration: none;
+   padding-top: 10px;
+   text-transform: uppercase;
+   
+ }
+ 
+ .artists a:visited {
+  color:  var(--primary);
+  opacity: 0.6;
+  text-transform: uppercase;
+  text-decoration: none;
+} 
+
+.artists > a:hover {
+  color: var(--third);
+  opacity: 1;
+}
+
+
+
+.artists > :not(:last-child):not(:first-child):before {
+  content: ", ";
+  white-space: pre;
+  color: var(--primary);
+}
+
+.artists > :last-child:before {
+  content: " & ";
+  white-space: pre;
+  color: var(--primary);
+}
+
+.artists > :only-child:before {
+  content: "";
+}
```

Comment:
This was such an ingenious snippet of code! And I would've never thought to do it in CSS either. Very well done Peki and Ajmal!